packages/preview2-shim/test/test.js

```diff
@@ -337,7 +337,9 @@ suite("Node.js Preview2", () => {
       mock.reset();
     });
 
-    test("tcp.connect(): should connect to a valid ipv4 address and port=0", async () => {
+    // test passing but blocked on socket.connect().
+    // TODO: figure out how to mock handle.connect()
+    test.skip("tcp.connect(): should connect to a valid ipv4 address and port=0", async () => {
```

Comment:
For these tests that can't be easily mocked, it would be better to remove them from this unit test suite and instead just rely on the integration test coverage since that is sufficient at this point.